### ISSUE TITLE
[INLONG-739] Create a temporary 0.10.0 development branch

### ISSUE THREAD

**Issue by dockerzhang** [apache/incubator-inlong#1328]
<p>The 0.9.0 version is currently being released, due to the possibility of the version being returned and resent, in order to carry out the next development in parallel, relevant improvements during the version release period are merged into this branch.</p>

<p> </p>

<p>After the version is released, the branch will be merged into the master for subsequent feature development by retaining detailed PR submission records</p>
<i>JIRA link - <a href="https://issues.apache.org/jira/browse/INLONG-739">[INLONG-739]</a> created by gosonzhang</i>
